README.md

```diff
@@ -68,13 +48,70 @@ To get started, copy the existing `env-example` file to `.env` inside the `insta
     - In case of a separately managed database setup, set the value to the database's hostname/IP/domain.
 - `SITES=site1.domain.com,site2.domain.com`
     - List of sites that are part of the deployment "bench". Each site is separated by a comma(,).
-    - If LetsEncrypt is being setup, make sure that the DNS for all the site domains are pointing to the current instance.
+    - If LetsEncrypt is being setup, make sure that the DNS for all the site's domains are pointing to the current instance.
 - `LETSENCRYPT_EMAIL=your.email@your.domain.com`
-    - Email for LetsEncrypt expiry notification. This is only required if you are setting up the nginx proxy companion.
+    - Email for LetsEncrypt expiry notification. This is only required if you are setting up LetsEncrypt.
+
+
+### Deployment for local development
```

Comment:
I think the current readme is pretty good else for the grammatical and the spacing changes that you have made.

Reply:
Most of the changes are reorganizing it so it's clearer and adding new sections, grammar and spacing is just a byproduct.

Reply:
I had troubles understanding the original readme, wasted many hours of work, all the details that were unclear to me (including those for which I opened issues) were clarified.

Reply:
But this section shouldn't exist. There's no "Deployment for local development" that is being provided through this docker. It's better to not misguide people on that. I hope it is understandable.

Reply:
These set of production images can be setup locally to make api calls from local non frappe app during development.

That needs port 80 to be exposed and makes the documentation relevant

To clear confusion may be mention somewhere about development/README.md